course/migrations/0025_auto_20210719_2232.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.0.14 on 2021-07-20 05:32
```

Comment:
This and all the above migrations that were added should be combined into a single migration. Also, note you will want to be up-to-date with master as more migrations have happened since then. Will help ensure the version numbering of the migrations is correct. Same as above we want a new feature implementation to only have a single migration.